app/layout.tsx

```diff
@@ -1,15 +1,24 @@
 'use client';
 // 팀 블로그 헤더
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import { Inter } from 'next/font/google';
 import { ThemeProvider } from 'styled-components';
 
+import Header from '@/components/common/Header';
 import { GlobalStyle } from '@/styles/GlobalStyle';
 import theme from '@/styles/theme';
 import StyledComponentsRegistry from '@/utils/lib/registry';
 
 const inter = Inter({ subsets: ['latin'] });
 
 export default function RootLayout({ children }: { children: React.ReactNode }) {
+  const queryClient = new QueryClient({
+    defaultOptions: {
+      queries: {
+        retry: 0,
+      },
+    },
+  });
```

Comment:
queryClient의 경우는 useState를 이용해서 리렌더 방지를 해줍시다! 고연산 작업이 필요한 친구니까용
```suggestion
  const [queryClient] = useState(new QueryClient({
    defaultOptions: {
      queries: {
        retry: 0,
      },
    },
  }));
```

Reply:
오호~ 이건 또 첨 알았네요 좀 더 찾아보니 넥제라서 필요한 작업이군요!-! 감사합니당 ~~